crates/ruff_linter/resources/test/fixtures/flake8_async/ASYNC100.py

```diff
@@ -1,51 +1,63 @@
 import anyio
 import asyncio
 import trio
+from contextlib import nullcontext
 
 
 async def func():
-    async with trio.fail_after():
```

Comment:
The `trio` and `anyio` context managers are not `async`. The difference is relevant for this change, so I changed them back to `with`, this way we test both `with` and `async with` (asyncio).

Reply:
Thanks, that's my bad!